src/api/pokemon.ts

```diff
@@ -0,0 +1,15 @@
+import {PokemonClient} from 'pokenode-ts';
+
+// getBerry returns berry info
+// TODO currently just prints stuff to console
+export const getPokemonList = async () => {
+  const api = new PokemonClient({
```

Comment:
eventually we'll probably want to pass our client into this function, that way we can create our api client only once and pass it into our functions.  (otherwise caching probably(???) wont work)